ups/meas_extensions_psfex.table

```diff
@@ -1,6 +1,6 @@
+setupRequired(meas_base)
 setupRequired(meas_algorithms)
 setupRequired(psfex)
-#setupRequired(pyfits)
```

Comment:
Strange, I don't have that on my local copy. I wonder If I forgot to push that change